src/yarpmanager/src-manager/applicationviewwidget.cpp

```diff
@@ -770,6 +763,14 @@ void ApplicationViewWidget::onModuleItemChanged(QTreeWidgetItem *it,int col)
     if (!(tmp & Qt::ItemIsEditable)) {
         return;
     }
+    if( col == 3 && it->text(col) == "localhost")
+    {
+        it->setTextColor(3,QColor("#A0A0A0"));
```

Comment:
My personal opinion is that the grey colour is wrong here. The user expects a "disabled" behaviour from the grey label.
Also forcing it manually will make it unreadable depending on the user colour scheme (just imagine if you have a dark theme and you set it to black

Reply:
I agree with you @drdanz for the "grey colour", the point is that I think we have to let the user know that running stuff on localhost is different to run on other hosts..